package.json

```diff
@@ -1,18 +1,18 @@
 {
-  "name": "phonegap-plugin-push",
-  "description": "Register and receive push notifications.",
+  "name": "ischemaview-phonegap-plugin-push",
```

Comment:
question: shouldn't the name be `@ischemaview/phone-gap-plugin-push`?

Reply:
I'm not quite sure which way is better. I followed the pattern from [cordova-plugin-background-download](https://github.com/ischemaview/cordova-plugin-background-download), which prepended `ischemaview` to the name of the package but did not include it in the repo. I wanted to stay consistent with the approach in that fork.

Reply:
got it 👍